src/apis/keycloak-api-service.ts

```diff
@@ -0,0 +1,90 @@
+import type { User } from "./schemas/keycloak/user";
+
+export interface KeycloakApiService {
+    getUsers: () => Promise<User[]>,
+    findUser: (id: string) => Promise<User[]>
+}
+
+/**
+ * Creates KeycloakApiService
+ */
+export function CreateKeycloakApiService(): KeycloakApiService {
+    const baseUrl: string = process.env.KEYCLOAK_BASE_URL
+    const realm: string = process.env.KEYCLOAK_REALM
+
+    return {
+        /**
+         * Gets all users from keycloak
+         * 
+         * @returns List of users
+         */
+        async getUsers(): Promise<User[]> {
```

Comment:
It looks like your indentation is set to 4 space or similar- please set to 2 spaces and update all files with updated indentation